src/content/docs/en/guides/testing.mdx

```diff
@@ -24,7 +24,7 @@ Playwright is an end-to-end testing framework for modern web apps. Use the Playw
 
 You can get started and run your tests using the [VS Code Extension](https://playwright.dev/docs/getting-started-vscode).
 
-Alternatively, you can install Playwright within your Astro project using the package manager of your choice. Follow the CLI steps to choose Javascript/Typescript, name your test folder, and add an optional Github Actions workflow.
+Alternatively, you can install Playwright within your Astro project using the package manager of your choice. Follow the CLI steps to choose Javascript/TypeScript, name your test folder, and add an optional Github Actions workflow.
```

Comment:
`Javascript` is also need fix.

There is a bit of a problem in the same position as the Chinese page `/zh-cn/guides/testing/`. Can you fix it?
```suggestion
Alternatively, you can install Playwright within your Astro project using the package manager of your choice. Follow the CLI steps to choose JavaScript/TypeScript, name your test folder, and add an optional Github Actions workflow.
```